### ISSUE TITLE
Control where .unison folder is placed?

### ISSUE THREAD

**Issue by delmartechdude** [bcpierce00/unison#160]
Hello, I'm trying to figure out how to change where the `.unison` folder is located. I found the "UNISON" environment variable reference, but that just says it lets you change the _name_ of the folder, not the _location_. I will be using similar config across Windows, Mac, and Linux, and I'd like the .unison folder to be placed and used in the same directory that the unison binary is located in.

**Comment by delmartechdude** [comment]
Nevermind again - my apologies. Looks like the "UNISON" environment variable not only changes the name, but also the location. A name on it's own, for example `export UNISON=mytestname` will place a folder in the same directory that the unison binary was run in, with that name. Doing something like `export UNISON=../mytestname2` will place that folder in the folder above where the Unison binary was run from. It would be *great* if we could integrate this into the documentation. "name" is not descriptive enough.

Tl;dr: Let's update the documentation to indicate that the UNISON environment variable can also change the location of the Unison config folder. Happy to make a pull request, unless there are particular ways that the documentation gets updated. Let me know how I can help.

**Comment by bcpierce00** [comment]
Please do make a PR.  

The only things to know about the documentation are that, (1) for profile / command-line preferences, the documentation lives in the .ml files where those preferences are defined, and (2) the rest of the documentation lives in a .tex file under /doc, not in src/strings.ml, which is automatically generated from the tex file.

    - B

**Comment by delmartechdude** [comment]
Just made a PR! Thanks @bcpierce00!
